testdata/scripts/func-merge-types.txt

```diff
@@ -0,0 +1,39 @@
+gofumpt -w .
```

Comment:
please just join all of these test scripts into something like `join-func-parameters.txt`. It is going to be shorter, because you can deduplicate stuff like `gofumpt -w`, `cmp`, `-- foo.go.golden --`, et cetera. The final file shouldn't be that large either, maybe 120 lines tops, which is OK. There aren't that many combinations we need to test for here.

there's also an overhead in splitting up these into tiny files. More subtests, more exec calls, etc. So we only want to separate stuff into multiple files when we are testing very different things. In this case, all of these are different inputs to the same feature, so I don't think it makes sense.

Reply:
As for the file that has the `skip`, you can just include that comment in `foo.go` commented out with a TODO.

Reply:
Done.